src/DropdownBase/DropdownBase.js

```diff
@@ -295,7 +296,7 @@ class DropdownBase extends React.PureComponent {
 
   _renderChildren() {
     const { children } = this.props;
-    const { selectedId } = this.state;
+    const { selectedId, open: isOpen } = this.state;
```

Comment:
If open can be undefined:
```suggestion
    const { selectedId, open } = this.state;
```